pinot-plugins/pinot-batch-ingestion/pinot-batch-ingestion-common/src/main/java/org/apache/pinot/plugin/ingestion/batch/common/SegmentGenerationTaskRunner.java

```diff
@@ -134,13 +135,15 @@ private SegmentNameGenerator getSegmentNameGenerator(SegmentGeneratorConfig segm
 
     boolean appendUUIDToSegmentName =
         Boolean.parseBoolean(segmentNameGeneratorConfigs.get(APPEND_UUID_TO_SEGMENT_NAME));
+    boolean excludeTimestampsInSegmentName =
```

Comment:
This can be applied to `NormalizedDateSegmentNameGenerator` as well, so I'd suggest adding it to avoid confusion

Reply:
does it make sense to add to the `NormalizedDateSegmentNameGenerator` though? it's specifically used for human-readable data strings in the segment name and the flag would remove them.

Reply:
Good point. Let's not add it then